src/languages/_french.json

```diff
@@ -702,7 +719,7 @@
         "no_accounts_found": "Aucun compte Ethereum n’a été trouvé"
       },
       "info": {
-        "address_copied_to_clipboard": "L’adresse a été copiée dans le presse-papiers"
+        "address_copied_to_clipboard": "L’adresse a été copiée dans le preefsse-papiers"
```

Comment:
idk french but wondering if this was accidental

Reply:
le i don't know